src/ServerSide.java

```diff
@@ -0,0 +1,87 @@
+import java.io.*;
+import java.net.*;
+
+
+public class ServerSide 
+{
+	public static void main(String[] args) throws Exception
+	{
+		System.out.println("Server");
+		// declaring and initializing required variables
+		int portNum = 9999, fileNameLen, fileContentLen, choice;
+		
+		ServerSocket serverSocketObj = new ServerSocket(portNum);
+		Socket serverSideSocket =  serverSocketObj.accept();
+		DataInputStream dataInputStreamObj = new DataInputStream(serverSideSocket.getInputStream());
+		
+		choice = dataInputStreamObj.readInt();
+		
+		if(choice == 1)
+		{
+			// receiving file
+			fileNameLen = dataInputStreamObj.readInt();
+			if(fileNameLen > 0)
+			{
+				// dealing with file title
+				byte[] fileNameInBytes = new byte[fileNameLen];
+				dataInputStreamObj.readFully(fileNameInBytes, 0, fileNameLen); // read data from input stream upTo given length
+				String fileName = new String(fileNameInBytes);
+//				
+				
+				// dealing with content of file
+				fileContentLen = dataInputStreamObj.readInt();
+				byte[] fileContentInBytes = new byte[fileContentLen];
+				dataInputStreamObj.readFully(fileContentInBytes, 0, fileContentLen);
+				
+				// creating a file at server end and writing it
+				File fileRecieved = new File( "C:\\Users\\Manish Sharma\\Downloads\\" + fileName);
+				FileOutputStream fileOutputStreamObj = new FileOutputStream(fileRecieved);
+				fileOutputStreamObj.write(fileContentInBytes);
+				fileOutputStreamObj.close();
+			}	
+		}
+		else // wrong choice is tackled at client side
```

Comment:
Try to write if and else in different function. Ideally all function should follow SRP principle which stand for Single responsibility principle. So try to create a function which do only one thing.